Formula/libbpg.rb

```diff
@@ -29,6 +29,8 @@ def install
     args << "USE_X265=" if build.without? "x265"
 
     system "make", "install", "prefix=#{prefix}", "CONFIG_APPLE=y", *args
+
+    share.install Dir["html/bpgdec*.js"]
```

Comment:
I don't really know where to put these files. They must be copied to a web site to enable viewing bpg images in your browsers.